INS_CURTAIN/src/features/curtain.go

```diff
@@ -78,36 +58,26 @@ func readImageFile(degree int) []byte {
 	}
 }
 
-// PrintClosedCurtain
-/*
-- Prints closed curtain.
-- Reads closed curtain image and prints it.
-- Parameter:
-  - none
-
-- Return Value: void
-	- It will do its task and exit the function.
-*/
-func PrintClosedCurtain() {
-	buf := readImageFile(0)
-	terminal.ClearTerminal() //* Clear the terminal first.
-	color.White(string(buf))
-	color.White(returnFormattedMsg(false, false, false, 0))
-}
-
-// PrintOpenedCurtain
+// PrintCurtain
 /*
-- Prints opened curtain.
-- Reads opened curtain image and prints it.
+- Prints curtain.
+- Reads curtain image and prints it.
 - Parameter:
-  - none
+  - degree: int
 
 - Return Value: void
 	- It will do its task and exit the function.
 */
-func PrintOpenedCurtain() {
-	buf := readImageFile(100)
-	terminal.ClearTerminal() //* Clear the terminal first.
-	color.HiBlack(string(buf))
-	color.HiBlack(returnFormattedMsg(false, false, false, 100))
+func PrintCurtain(degree int) {
+	if degree < 50 { // closed(down)
```

Comment:
Merged two different functionality into one single function. Affordable, since it still has only one purpose at single function (Print Curtain) in a broad perspective.